packages/sender/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "forceConsistentCasingInFileNames": true,
     "strict": true,
     "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "resolveJsonModule": true,
```

Comment:
Is this needed for this PR? Just curious, I didn't notice any JSON file imports

Reply:
This is needed to be done in order to get the sender test running. Without this, upon running the sender test, it complains about undefined path on `en.json`

Reply:
Good to know, thanks!